setup.py

```diff
@@ -2,17 +2,33 @@
 
 from distutils.core import setup
 
+#try:
+#    import pypandoc
+#    long_description = pypandoc.convert('README.md', 'rst')
+#except(IOError, ImportError):
+#    long_description = open('README.md').read()
+
 setup(
-    name='dali',
+    name='python-dali',
     version='0.3',
     description='Interface to DALI lighting systems',
     author='Stephen Early',
     author_email='steve@assorted.org.uk',
     url='https://github.com/sde1000/python-dali',
+    download_url='https://github.com/dgomes/python-dali/archive/python-dali-0.3.tar.gz',
     packages=[
         'dali',
         'dali.device',
         'dali.driver',
         'dali.gear'
     ],
+    classifiers=[
+        'Development Status :: 4 - Beta',
```

Comment:
Well I can agree on that :)

I went with Beta since it actually fitted 100% of my use case, but point taken